include/geode/model/representation/builder/brep_builder.h

```diff
@@ -80,12 +82,12 @@ namespace geode
     public:
         struct ComponentMapping
```

Comment:
This struct may be moved out from BrepBuilder. I think define an absl::flat_hash_map< geode::ComponentType, UUIDMapping >: it may allows to keep separated mappings (as in the old version and not in the new one for collections), but it is extensive